src/backend/base/langflow/graph/graph/state_manager.py

```diff
@@ -33,15 +33,3 @@ def get_state(self, key, run_id: str):
 
     def subscribe(self, key, observer: Callable) -> None:
         self.state_service.subscribe(key, observer)
-
-    def notify_observers(self, key, new_state) -> None:
```

Comment:
not used and incorrect (self.observers doesn't exist)

Reply:
LGTM